src/lib.rs

```diff
@@ -774,3 +774,13 @@ impl Mode {
         }
     }
 }
+
+impl Display for Mode {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Mode::Pass => write!(f, "pass"),
+            Mode::Panic => write!(f, "panic"),
+            Mode::Fail{ require_patterns: _ }  => write!(f, "fail ")
```

Comment:
Why does this have a trailing space, but the other two do not?

Reply:
Because I did not run rustfmt. Done now!